src/Entity/Product.php

```diff
@@ -88,27 +87,27 @@ public function setPrice(string $price): self
     /**
      * @return Collection<int, Orders>
      */
-    public function getProd(): Collection
+    public function getProduct(): Collection
     {
-        return $this->prod;
+        return $this->product;
     }
 
-    public function addProd(Orders $prod): self
+    public function addProduct(Orders $product): self
     {
-        if (!$this->prod->contains($prod)) {
-            $this->prod[] = $prod;
-            $prod->setProductId($this);
+        if (!$this->product->contains($product)) {
+            $this->product[] = $product;
+            $product->setProductId($this);
         }
 
         return $this;
     }
 
-    public function removeProd(Orders $prod): self
+    public function removeProd(Orders $product): self
```

Comment:
Method name can be renamed to `removeProduct`

Reply:
renamed method name